.github/workflows/msys2.yml

```diff
@@ -0,0 +1,76 @@
+name: msys2
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+
+jobs:
+  msys2:
+    runs-on: windows-latest
+
+    steps:
+    - uses: msys2/setup-msys2@v2
+
+    # pre-requires
+    - name: Install dependencies
+      shell: msys2 {0}
+      run: |
+        # pacman -Syu --noconfirm # Fail, requires to close this process
+        pacman -S --noconfirm mingw-w64-clang-x86_64-cmake mingw-w64-clang-x86_64-make mingw-w64-clang-x86_64-clang mingw-w64-clang-x86_64-clang-tools-extra
+        pacman -S --noconfirm mingw-w64-clang-x86_64-zlib mingw-w64-clang-x86_64-libssh mingw-w64-clang-x86_64-hunspell mingw-w64-clang-x86_64-openssl mingw-w64-clang-x86_64-sqlite3
+
+    # WxWidgets
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        repository: wxWidgets/wxWidgets
+        path: wxWidgets
+        # ref: 3.2.4 # use master
```

Comment:
Cannot use tagged version of WxWidget :/

> D:\a\codelite\codelite\Plugin\drawingutils.cpp:409:48: error: no member named 'SelectLightDark' in 'clSystemSettings'
  409 |     wxColour darkPen = clSystemSettings::Get().SelectLightDark(bgColour.ChangeLightness(95), *wxBLACK);